libevm/VMConfig.h

```diff
@@ -25,9 +25,13 @@ namespace eth
 //
 // interpreter configuration macros for development, optimizations and tracing
 //
+// EIP_145                - bitwise shifting
```

Comment:
Out of curiosity, why is `EIP_145` better/more expressive than `EVM_USE_BITSHIFT` ?

Reply:
It tells the reader where to find the spec for the feature.  And it's consistent with the others.

Reply:
I prefer using a meaning names, because I'm usually confused by numbers.
As a compromise, can it be `EIP145_BITSHIFT` or `EVM_EIP145_BITSHIFT`?

Reply:
I much prefer the compromise.